scylla-server/tests/data-services.test.ts

```diff
@@ -39,4 +40,34 @@ describe('Data', () => {
       `Invalid data provided, Expected data of type DataTypeName and got {"bruh":"test"}`
     );
   });
+
+  // test('Add Data', async () => {
+  //   const expected = ['test'];
+
+  //   const serverData: ServerData = {
+  //     name: 'test',
+  //     value: 0,
+  //     units: 'lbs'
+  //   };
+
+  //   await DataService.addData(serverData, 1, 1, 1);
+
+  //   const data = {
+  //     dataTypeName: 'test'
+  //   };
```

Comment:
when you do this your gonna have to do some fancy delete stuff like how i did it in the getAllRuns PR